public/components/agents/vuls/inventory/detail.tsx

```diff
@@ -0,0 +1,381 @@
+            return 0;
+          }
+        })
+        .map(({ username, permissions }) => {
+          return (
+            <EuiToolTip
+              key={`permissions-windows-user-${username}`}
+              content={permissions.join(', ')}
+              title={`${username} permissions`}
+            >
+              <EuiBadge color="hollow" title={null} style={{ margin: '2px 2px' }}>
+                {username}
+              </EuiBadge>
+            </EuiToolTip>
+          );
+        });
+      return (
+        <TruncateHorizontalComponents
+          components={components}
+          labelButtonHideComponents={(count) => `+${count} users`}
+          buttonProps={{ size: 'xs' }}
+          componentsWidthPercentage={0.85}
+        />
+      );
+    }
+    return value;
+  }
+
+
+  render() {
+    const { type, implicitFilters, view, currentItem, agent } = this.props;
+    const id = `${currentItem.name}-${currentItem.cve}-${currentItem.architecture}-${currentItem.version}`;
+    const inspectButtonText = view === 'extern' ? 'Inspect in Dashboard' : 'Inspect in Events';
+    return (
+      <Fragment>
+        <EuiAccordion
+          id={id === undefined ? Math.random().toString() : `${id}_details`}
+          buttonContent={
+            <EuiTitle size="s">
+              <h3>Details</h3>
+            </EuiTitle>
+          }
+          paddingSize="none"
+          initialIsOpen={true}
+        >
+          <div className="flyout-row details-row">{this.getDetails()}</div>
+        </EuiAccordion>
+        <EuiSpacer />
+        <EuiAccordion
+          id={id === undefined ? Math.random().toString() : `${id}_events`}
+          className="events-accordion"
+          extraAction={
+            <div style={{ marginBottom: 5 }}>
+              <strong>{this.state.totalHits || 0}</strong> hits
+            </div>
+          }
+          buttonContent={
+            <EuiTitle size="s">
+              <h3>
+                Recent events
+                {view !== 'events' && (
+                  <span style={{ marginLeft: 16 }}>
+                    <EuiToolTip position="top" content={inspectButtonText}>
+                      <EuiIcon
+                        className="euiButtonIcon euiButtonIcon--primary"
+                        onMouseDown={(ev) => this.viewInEvents(ev)}
+                        type="popout"
+                        aria-label={inspectButtonText}
+                      />
+                    </EuiToolTip>
+                  </span>
+                )}
+              </h3>
+            </EuiTitle>
+          }
+          paddingSize="none"
+          initialIsOpen={true}
+        >
+          <EuiFlexGroup className="flyout-row">
+            <EuiFlexItem>
+              <Discover
+                kbnSearchBar
+                shareFilterManager={this.discoverFilterManager}
+                initialColumns={[
+                  'icon',
+                  'timestamp',
+                  // 'rule.mitre.id',
```

Comment:
If this field is not necessary, you could remove it.

Reply:
removed!